doc/cug.tex

```diff
@@ -2776,25 +2776,29 @@ \subsubsection{Graph Types}
 \label{fig-test2}
 \end{figure}
 
-\subsubsection{Graph Headers}
+\subsubsection{Graph Section Headings}
 
-In the above examples graph headings are used to define recurring date-times
-when tasks are to run. For example in the following scenario,
+Graph section headings define recurrence expressions, the graph within a graph
+section header defines a workflow at each point of the recurrence. For
```

Comment:
(Sorry to be pendantic, but!) can we have consistent use of "heading" rather than "header" throughout.